MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include config/*
+include release/*
```

Comment:
So I think one thing we could do to avoid having to wrestle with where the files are is to have config.json and the data files in release, like those in samples, _not_ be in the package (and so not in the manifest).  This way they'd just be auxiliary files that you'd use but not install...

Reply:
So, should I just put files in `config` and `data` into `release`? And then get rid of the manifest?

Reply:
Something like that - I'd say that "data" is probably a better name than "release", so fold them into there and then yes either remove the manifest or use it to explicitly exclude `data`, whichever is easier.